qcodes/tests/dataset/legacy_DB_generation/utils.py

```diff
@@ -1,11 +1,26 @@
 # General utilities for the database generation and loading scheme
-from typing import Dict, List
+from typing import Dict, List, Tuple
 from contextlib import contextmanager
+import os
 
-GIT_HASHES: Dict[int, str] = {0: '78d42620fc245a975b5a615ed5e33061baac7846'}
+from git import Repo
+
+# A brief overview of what each version introduces:
+#
+# Version 0: the original table schema, runs, experiments, layouts,
+# dependencies, result-tables
+#
+# Version 1: a GUID column is added to the runs table
```

Comment:
is description of version 2 missing?

Reply:
We will add that when we need it, i.e. when we have version 3 on the table.